Review of the Logging guide (#52934)
quarkusio/quarkus: docs/src/main/asciidoc/logging.adoc

   }
}
----
<1> The fully qualified class name (FQCN) of the declaring class is used as the logger name.
For example, `org.jboss.logging.Logger.getLogger(SimpleBean.class)` is used.

> <2> In this case, the name _foo_ is used as a logger name, for example, `org.jboss.logging.Logger.getLogger("foo")` will be used.

Reviewer: ```suggestion
<2> In this case, the name _foo_ is used as a logger name.
The logger is created using `org.jboss.logging.Logger.getLogger("foo")`.
```

        level: DEBUG
----

> 

Author: Seems like some history leftover problem. The PR is rebased and force-pushed; yet this historical state is still here.

When you set the log level to below `DEBUG`, for example `TRACE`, also adjust the minimum log level.
Set the minimum log level globally with the `quarkus.log.min-level` configuration property, or set it per category:

[source, properties]
----
quarkus.log.category."org.hibernate".min-level=TRACE
----

> This sets a floor level that Quarkus needs to generate supporting code for.

Reviewer: This is harder to read than before.

The minimum log level must be set at build time so that Quarkus can open the door to optimization opportunities where logging on unusable levels can be elided.

.An example from a native executable:
Setting `INFO` as the minimum log level sets lower level checks, such as `isTraceEnabled()`, to `false`.

> This identifies code such as `if (logger.isDebugEnabled()) callMethod();` that never runs and marks it as dead code.

Author: THX for the great feedback. Applying.
